<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Channel</class>
 <widget class="QWidget" name="Channel">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>324</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="next_turn_button">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Prochain tour</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="end_round_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Fin du round</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="channel_textEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
